account/utils.py

```diff
@@ -0,0 +1,31 @@
+import json
+import requests
+import jwt
+
+from django.http import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from chanel.settings import SECRET_KEY, ALGORITHM
+from .models import Account
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        token = request.headers.get('Authorization', None)
+        
+        if token is not None:
+
+            try:
+                payload          = jwt.decode(token, SECRET_KEY, algorithm=ALGORITHM)
+                userinfo         = Account.objects.get(email=payload['email'])
+                request.userinfo = userinfo
+
+                return func(self, request, *args, **kwargs)
+
+            except jwt.exceptions.DecodeError:
+                return JsonResponse({'message' : 'INVALID_TOKEN' }, status=400)
+
+            except Account.DoesNotExist:
+                return JsonResponse({'message' : 'This User Does Not Exist'}, status=400)
+        else:
+            return JsonResponse({'message' : "You need to log in first"}, status=401)
```

Comment:
`else`가 없더라도 위의 `if` 조건이 충족되지 않으면 도달 할 수 있는 부분입니다. `else` 제거하시는 것을 추천드립니다.
메세지도 보다 간략하게 수정해주세요. 해당 메세지는 고객이 읽는 것이 아니라 프론트엔드 개발자가 보는 것이므로 명료한 편이 나을 것 같습니다.

따옴표 또한 통일해주세요~
```suggestion
        
        return JsonResponse({'message' : 'LOGIN_REQUIRED'}, status=401)
```